src/app/client/src/app/modules/dashboard/components/addusers/addusers.component.ts

```diff
@@ -4487,19 +4487,19 @@ export class AddusersComponent implements OnInit {
       if (this.roleEditeUserData.organisations.length == 1) {
         this.addRolePopup = true
         this.rootAdminOrgList();
-        this.roleEditData = this.roleEditeUserData.organisations[0].roles;
+        this.roleEditData = this.roleEditeUserData.roles;
         this.addRoleorgId = this.roleEditeUserData.organisations[0].organisationId;
```

Comment:
@snehalsabade Have you confirmed the value of 'addRoleorgId'  this variable?

Reply:
Yes mam